components/generic/Tabs.tsx

```diff
@@ -38,7 +38,7 @@ export default function Tabs({ tabs, children }: { tabs: ITab[]; children: React
             <div className=" -mb-px flex justify-start">
               {tabs.map((tab) => (
                 <Link
-                  href={tab.href}
```

Comment:
can you try without the function `pathChangeTail`? it just works for me.

Reply:
without `pathChangeTail` user pushes tabs `ROLES` being on e.g. `Developers` or any other  and users is pushed to `Admins`

Reply:
@Kolezhanchik sorry, not following you comment, would you mind give me little more details with actual URLs?

Reply:
when user pushes tab `ROLES` on product page tabs user is routed to 
![image](https://github.com/bcgov/platform-services-registry/assets/65827870/48f94076-6dcb-452b-a23a-461b2c11d3b7)
/public-cloud/aws-roles/eu9cfk/developers

user being in tab  `ROLES` pushes tab `Developers` and is routed to /public-cloud/aws-roles/eu9cfk/developers
![image](https://github.com/bcgov/platform-services-registry/assets/65827870/b3880e49-6c12-4d58-94c9-234bdf159609)

being in `ROLES` + `Developers` user can push `ROLES`(which is clickable)  and this user is routed to `Admins`, which is not expected(expected to stay on `Developers` tab

Reply:
@Kolezhanchik I think I am getting it and we need to identify that it is actually a bug though. I see the ticket you are working on and it mentions of the `ROLES tab not highlighted` as the bug.
So, do you think it is a bug? to me it is an expected behaviour as the `Admins` tab is the default tab of the upper level `ROLES` tab.

Reply:
removed `pathChangeTail` from Tabs